geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -202,6 +202,30 @@ public void zremCanRemoveMembersToBeRemoved() {
     verify(sortedSet).storeChanges(eq(region), eq(key), any(RemsDeltaInfo.class));
   }
 
+  @Test
+  public void zremrangebylex_removesMembersInRange() {
```

Comment:
I'm not certain that this test is needed, since the behaviour should be covered extensively in the integration test.